scripts/build-engine/src/index.ts

```diff
@@ -161,6 +161,11 @@ namespace build {
         };
 
         buildTimeConstants: IBuildTimeConstants;
+
+        /**
+         * Whether force SUPPORT_JIT to the specified value.
+         */
+        forceJitValue?: boolean,
```

Comment:
What does this variable mean? Force means should support JIT no matter it supports or not.

Reply:
add a option in params, openharmony platform will pass a false, disable jit。

Reply:
What does it be used?

Reply:
Turn off the jit optimization of the javascript layer